docs/source/notebooks/README.ipynb

```diff
@@ -15,7 +15,9 @@
     "\n",
     "These servers are intended to be accessed through a programming environment. More precisely, analytical services can be accessed through a Web Processing Services (WPS) Application programming interface (API). To facilitate the integration of these WPS processes into typical scientific workflows, we suggest using the high-level Python client library (`birdy.WPSClient`). \n",
     "\n",
-    "This JupyterLab is not meant as a production environment. It is a public server with no private directory. Your files are public, can be seen by the rest of the world and can be reset at anytime. Use it to explore the system's capability and see whether it may be useful to your work. If it is, install the `birdy` client locally along with software to read netCDF files and you're set to go. \n",
+    "The `public` demo account on this JupyterLab is not meant as a production environment. It is a public account with no private directory and limited computing resources. Your files are public, can be seen by the rest of the world and can be reset at anytime. Use it to explore the system's capability and see whether it may be useful to your work.\n",
+    "\n",
+    "You can request personal account by contacting the email on the login page.  With a personal account, you will have a production environment with private workspace and no computing resource limitation.\n",
```

Comment:
Once logged, you need to log out to see the login page again. I suggest we repeat the email here. Also, do we give an account to everyone, do we ask for some kind of description of their purpose ? I'm not suggesting we do, just asking.

Reply:
>  I suggest we repeat the email here

The email on the login page is customizable through `env.local` file.  Repeating the email here means all the other organizations deploying PAVICS will contact us :(